.github/workflows/build-nym-vpn-app-linux.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         with:
           node-version: 20
 
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
```

Comment:
Imho let's stick to the version that core uses

Reply:
ok please push to the pr the version you want

Reply:
as side note, using latest is building -> https://github.com/nymtech/nym-vpn-client/actions/runs/11495747973